.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+###### Description
+
+
+###### System Info
+macOS 10.
+Xcode 8.
+
+###### Verification
+- [ ] Have you checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
+- [ ] Have you referenced relating tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
+- [ ] Have you checked your Portfile with `port lint`?
+- [ ] Have you tried existing tests with `sudo port test`?
+- [ ] Have you tried a full install with `sudo port install`?
```

Comment:
`sudo port install` vs. `destroot`: I have no preference. I would probably add the flags `-v -t`. Generally developers need to install the port anyway in order to be able to test it, so asking for `destroot` would be asking them to do less (if they are too lazy, they'll know they could do destroot themselves).